plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/editor/tabbar/actions/TabbarPasteFormatMenuManager.java

```diff
@@ -69,6 +69,10 @@ protected void itemRemoved(IContributionItem item) {
     @Override
     public void dispose() {
         removeAll();
+        if (actionHistory != null) {
+            actionHistory.clear();
+            actionHistory = null;
+        }
```

Comment:
just actionHistory = null is enough because removeAll will clear the list and dispose all action

Anyway is it really usefull? because the instance will be garbage colleted later and all stuff that needed to be disposed has been disposed.

Reply:
OK for just "actionHistory = null"

Useful ? A test has been added to detect this case of problem. So probably to address a potential leak (actions retained by the new array list and never garbage collected...). But not sure.